### PR TITLE
OLMIS-7772: Fix issues with Chrome popover functionality

### PR DESCRIPTION
**Pull request by dariapionk**

### CODE REVIEW

File: src/stock-card-summary-list/stock-card-summary-list.html [hunk 1]
```diff
@@ -85,6 +85,6 @@ <h2 ng-hide="vm.facility && vm.program">{{'stockCardSummaryList.stockOnHand' | m
             </tr>
         </tbody>
     </table>
-    <openlmis-pagination ng-if="!vm.offline()" list="vm.displayStockCardSummaries" paged-list="vm.pagedList" pagination-id="'stockCardSummaries'"/>
+    <openlmis-pagination ng-if="!vm.offline()" list="vm.displayStockCardSummaries" paged-list="vm.pagedList" pagination-id="'stockCardList'"/>
```
Comment (kmadej) [inline src/stock-card-summary-list/stock-card-summary-list.html:88]: should it be in scope of this PR ?

**Review by kmadej — CHANGES_REQUESTED** [review]

File: src/stock-card-summary-list/stock-card-summary-list.html (hunk 1 above)
Comment (dariapionk) [reply src/stock-card-summary-list/stock-card-summary-list.html:88]: It is not related, however it's an old bug which should have been corrected some time ago - ticket for it is not created, so I thought that we can resolve this bug now

Comment (dariapionk) [reply src/stock-card-summary-list/stock-card-summary-list.html:88]: I can delete this line if needed

**Review by kmadej — APPROVED** [review]
Looks good to me

**Pull request merged**